cups/auth.c

```diff
@@ -330,24 +330,6 @@ _cupsSetNegotiateAuthString(
   gss_buffer_desc output_token = GSS_C_EMPTY_BUFFER;
 					/* Output token */
 
-
-  (void)method;
-  (void)resource;
-
-#  ifdef __APPLE__
- /*
-  * If the weak-linked GSSAPI/Kerberos library is not present, don't try
```

Comment:
This and the other Apple-specific NULL checks cannot be removed because it will break things (these libraries are weak-linked and so might not always be available...)